### ISSUE TITLE
Partners on world map

### ISSUE THREAD

**Issue by dianaszpotowicz** [godanSec/Website#31]
Hi,

I'm wondering if all the partners are showing up on the world map?

For example, when you first go to the partners page, it shows that we don't have any partners from Canada. But we have 2 partners on there. They show up when you specifically search for 'Canada' but not on the original 'worldview' map.

Just curious if this might be the case for other regions as well?

**Comment by VTsantilis** [comment]
Yes they are properly shown, but the marker clustering effect groups all markers that are close to each other in combination with the zoom level. If you mouseover the circles shown on map, you can see a blue border showing the area for the grouped markers. When someone clicks the circle, map is zoomed and markers are detached. You can zoom in - out using the map controls to see how the clustering behaves. It is a nice feature but if you think it's confusing we could disable it.

**Comment by dianaszpotowicz** [comment]
Thank you - but indeed not all the markers are shown. For example we have
partners in Canada that do not show up on the map at all when you first
click on the partners page.

On Friday, 8 April 2016, VTsantilis notifications@github.com wrote:

> Yes they are properly shown, but the marker clustering effect groups all
> markers that are close to each other in combination with the zoom level. If
> you mouseover the circles shown on map, you can see a blue border showing
> the area for the grouped markers. When someone clicks the circle, map is
> zoomed and markers are detached. You can zoom in - out using the map
> controls to see how the clustering behaves. It is a nice feature but if you
> think it's confusing we could disable it.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/godanSec/Website/issues/31#issuecomment-207488629

##

**Comment by stoitsis** [comment]
@dianaszpotowicz  The partners in Canada are clustered in the North East America cluster that shows 50. We can try to adjust a bit more the clustering setting and as Vassilis @VTsantilis says we could even disable it if it confuses the user.

**Comment by anabrandusescu** [comment]
@dianaszpotowicz @stoitsis @VTsantilis I suggest to try out the following: uncluster the partners so they are simply point based. Do you have some fancy features for pins? Something close to what we had on our old map? (e.g. 70% opaque green circles: https://godan.cartodb.com/viz/e0e32610-c9ae-11e5-8c92-0ef7f98ade21/map
![point-map](https://cloud.githubusercontent.com/assets/3058770/14524242/b7278be2-0237-11e6-830e-1c6405c8c913.png)

**Comment by stoitsis** [comment]
@anabrandusescu the cartodb version is fancy but not very user friendly e.g. there are many overlaps in the markers that makes the navigation hard. What we can do is a) to reduce the clustering as we already did in the live site http://www.godan.info/partners and b) to add a GODAN custom marker like the one in the screen
![screen shot 2016-04-19 at 2 30 22 pm](https://cloud.githubusercontent.com/assets/3452880/14637693/a4ddf658-063b-11e6-964f-051885adcd4a.png)

If we want to have clustering per country then we could use a plugin http://regionbound.com/ that costs 39$. Let us know what do you think.

**Comment by anabrandusescu** [comment]
@stoitsis ok - less clustering is better but I'd recommend even less clustering if possible. Could we have a hover over function to click over points? That way the user doesn't have to go to on the ground zoom level when clicking to see the partner. And great idea about a GODAN custom marker - please add this feature

**Comment by dianaszpotowicz** [comment]
Yes – I think it’s very important to adjust the cluster setting. Our partnership in Canada, for example, is very important and its important the website user can see this right away. Right now it takes two ‘zoom in’ clicks before the user can have an idea that we have partnerships in Canada… there is the possibility they could be upset to be lumped into American partnerships.

Kind regards,

Diana Szpotowicz

Strategic Communications Advisor 

Global Open Data for Agriculture and Nutrition (GODAN)

Diana.Szpotowicz@godan.info

+44 01491 829315

@godanSec

**Comment by stoitsis** [comment]
@dianaszpotowicz we have prepared a new version with less clustering and clustering per country in http://dev-godan-new.pantheonsite.io/partners. @anabrandusescu a hover over function is not possible with clustering. You can use the toggle button on top right of the map to see how the maps looks without clustering. We suggest to keep the clustering as without it the navigation is hard.
